src/Home/index.js

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Map from '../components/Map';
 import VerticalToolBar from '../components/VerticalToolBar';
 import CreateButton from '../components/CreateNew';
 import ApolloAppBar from '../components/AppBar';
 import { Row } from 'mui-flex-layout';
+import Modal from '../components/Modal';
 
 export default () => {
+  const [openedModal, setOpenedModal] = useState(false);
+
+  const openModal = () => {
+    console.log('open');
+    setOpenedModal(true);
+  };
+
+  const closeModal = () => {
+    console.log('close');
+    setOpenedModal(false);
+  };
   return (
     <Row width={'100%'} height={'100%'}>
       <ApolloAppBar />
       <Map />
       <VerticalToolBar />
-      <CreateButton />
+      <CreateButton onClick={() => openModal()}/>
```

Comment:
im working on create page so put it on another button, where you really want the feature to be